src/index.css

```diff
@@ -46,3 +46,24 @@ a {
   border-left: 1.5px solid #000;
   height: 15px;
 }
+
+.styled-link {
+  text-decoration: underline;
+  font-weight: bold;
+}
+
+.no-styled-link {
+  text-decoration: none;
+  font-weight: normal;
+}
```

Comment:
Please remove these lines and just add  `.active { 
text-decoration: underline }` in order to style the links in a short and simple way.